server/src/main/java/org/elasticsearch/action/search/SearchRequest.java

```diff
@@ -125,7 +125,7 @@ public SearchRequest(String... indices) {
     /**
      * Constructs a new search request against the provided indices with the given search source.
      */
-    public SearchRequest(String[] indices, SearchSourceBuilder source) {
+    public SearchRequest(String... indices, SearchSourceBuilder source) {
```

Comment:
Does this compile for you? A vararg parameter should be the last one, which is why indices is an array here.

Reply:
Thank you for answering.
No, it's not compilation for myself.

As we will see later, we changed it because it can not be instantiated in the same way.
It was nice to hear the reason.
This is not a big fix, so this pull request will end.

following writing is possible.
```
String index = "index_name";
SearchRequest se = new SearchRequest(index );
```

following writing is not possible.
```
String index = "index_name";
SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder(); 
searchSourceBuilder.query(QueryBuilders.matchAllQuery()); 

// fail with the following
SearchRequest se = new SearchRequest(index, searchSourceBuilder );
```